tests/utils/test_image_extraction.py

```
@@ -52,19 +56,56 @@
+        Path(rootfs),
+        crane_bin=crane,
+        tar_bin=tar,
+        max_attempts=1,
+    )
+
+
+def _terminate_export_process_group(process) -> None:
+    try:
+        os.killpg(process.pid, signal.SIGTERM)
+    except ProcessLookupError:
+        pass
```

Ruff (0.16.1)

[warning] Use `contextlib.suppress(ProcessLookupError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(ProcessLookupError): ...`

(SIM105)